.pylintrc

```diff
@@ -12,7 +12,10 @@ disable =
 	bad-continuation,
 	too-few-public-methods,
 	global-statement,
+	unnecessary-comprehension,
```

Comment:
Did we need to add more to the ignore? I'd rather fix things to clean up the code as pylint introduces new checks as much as possible, rather than ignoring things.

Reply:
The top level imports I understand, though if it's only one instance I'd prefer it be a local override so that we get warnings in the future. 

What prompted ignoring unnecessary-comprehension and no-else-continue?

Reply:
I looked at the spots it was warning us and didn't see a simple way to fix the code. I wasn't going to spend a ton of time on it. If you'd rather I can ignore each line/file instead. But I will ignore it.

Reply:
Most of the fixes were just removing the comprehension. Most of it was on `<ldap group>.get_members()`, which already returns a list.